File: src/main/kotlin/stellarwitch7/ram/spell/trick/blunder/CellNotWithinWorldBlunder.kt
package stellarwitch7.ram.spell.trick.blunder

import dev.enjarai.trickster.spell.trick.Trick
import dev.enjarai.trickster.spell.trick.blunder.TrickBlunderException
import net.minecraft.text.MutableText

class CellNotWithinWorldBlunder(source: Trick) : TrickBlunderException(source) {
    override fun createMessage(): MutableText {
        return super.createMessage().append("Cell cannot be accessed as it does not exist")
    }
}

File: src/main/scala/stellarwitch7/ram/spell/blunder/CellNotWithinWorldBlunder.scala
package stellarwitch7.ram.spell.blunder

import dev.enjarai.trickster.spell.trick.Trick
import dev.enjarai.trickster.spell.blunder.TrickBlunderException
import net.minecraft.text.MutableText

class CellNotWithinWorldBlunder(source: Trick) extends TrickBlunderException(source) {
    override def createMessage(): MutableText =
      super.createMessage().append("Cell cannot be accessed as it does not exist")
}
